python/mxnet/contrib/onnx/onnx2mx/_op_translations.py

```diff
@@ -209,7 +211,9 @@ def batch_norm(attrs, inputs, proto_obj):
                                                                'is_test': 'fix_gamma'})
     new_attrs = translation_utils._remove_attributes(new_attrs,
                                                      ['spatial', 'consumed_inputs'])
-    new_attrs = translation_utils._add_extra_attributes(new_attrs, {'cudnn_off': 1})
+    cudnn_eps = get_cudnn_epsilon()
+    cudnn_off = 0 if not math.isnan(cudnn_eps) and attrs['epsilon'] >= cudnn_eps else 1
```

Comment:
if ``attrs`` does not contain 'epsilon', then it should be defaulted to 1e-5. ([reference](https://github.com/onnx/onnx/blob/master/docs/Operators.md#attributes-3))

Reply:
Good point!

Reply:
@anirudhacharya I just updated the PR. Let me know if you need anything else.